src/common/options/rgw.yaml.in

```diff
@@ -3556,3 +3556,113 @@ options:
   - rgw
   - osd
   with_legacy: true
+- name: rgw_directory_address
```

Comment:
I think we can cut all of these config variables for now since we're not using them?